examples/advanced/flutter/LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
Is this a placeholder file that someone who generates flutter would need to fill out before distribution of the generated assets? If so, it might be good to give them a bit of description to tell them the purpose of the file / be explicit in what to do. Maybe this is well known to flutter folk, but to me I was unsure if this was for us TODO.

If this is not a placeholder file for flutter users, lets delete it.

Reply:
This is also a placeholder, I will delete it since this flutter project is part of style-dictionary, and should follow it's license.